compiler/ballerina-compiler-api/build.gradle

```diff
@@ -27,6 +27,9 @@ dependencies {
 
     testCompile 'org.testng:testng'
     testCompile project(path: ':ballerina-test-utils', configuration: 'shadow')
+    testCompile project(':ballerina-io')
+
+    baloTestImplementation project(path: ':ballerina-io', configuration: 'baloImplementation')
```

Comment:
Do we still need this?

Reply:
No we don't. Not sure how this came up here again. Will remove this.